### PR TITLE
Update XML references wrt XI 11.12 (d15-7 with xcode9.4 support) and bump to 11.13

### PR DESCRIPTION
**Pull request by spouliot**

### CODE REVIEW

File: tools/apidiff/.vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (rolfbjarne) [inline tools/apidiff/.vscode/launch.json]: Did you mean to include this file?

**Comment by rolfbjarne** [comment]
Looks like I'll be fixing https://github.com/xamarin/maccore/issues/830 quite soon...

File: tools/apidiff/.vscode/launch.json (hunk 1 above)
Comment (spouliot) [reply tools/apidiff/.vscode/launch.json]: no :)

**Comment by spouliot** [comment]
build

**Comment by spouliot** [comment]
@rolfbjarne only the XI version number changed, so that should not 🔥 a breaking change warning

**Comment by spouliot** [comment]
and _ideally_ the *please review changes* for the generator should be skipped (since that will always change even if the API are identical).

**Comment by spouliot** [comment]
@rolfbjarne oh, it's a `const` so it make sense (from API diff point of view)

**Pull request merged**